src/pymor/reductors/ph/ph_irka.py

```diff
@@ -0,0 +1,90 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+from pymor.models.iosys import PHLTIModel
+from pymor.reductors.h2 import GenericIRKAReductor, OneSidedIRKAReductor
+from pymor.reductors.ph.basic import PHLTIPGReductor
+
+
+class PHIRKAReductor(GenericIRKAReductor):
+    """PH-IRKA reductor.
+
+    Parameters
+    ----------
+    fom
+        The full-order |PHLTIModel| to reduce.
+    mu
+        |Parameter values|.
+    """
+
+    def __init__(self, fom, mu=None):
+        assert isinstance(fom, PHLTIModel)
+        super().__init__(fom, mu=mu)
+
+    def reduce(self, rom0_params, tol=1e-4, maxit=100, num_prev=1, projection='orth', conv_crit='sigma',
+               compute_errors=False):
+        r"""Reduce using PH-IRKA.
+
+        It uses IRKA as the intermediate reductor.
+
+        See :cite:`GPBV12`.
+
+        Parameters
+        ----------
+        rom0_params
+            Can be:
+
+            - order of the reduced model (a positive integer),
+            - dict with `'sigma'`, `'b'`, `'c'` as keys mapping to
+              initial interpolation points (a 1D |NumPy array|), right
+              tangential directions (|VectorArray| from
+              `fom.D.source`), and left tangential directions
+              (|VectorArray| from `fom.D.range`), all of the same
+              length (the order of the reduced model),
```

Comment:
This part also needs to be updated.

Reply:
There should also be a point about only giving initial interpolation points. Or, it might be easiest to link to the docs of `OneSidedIRKAReductor`.

Reply:
Can I do this in a nice way?

Reply:
Thinking about it again and the point below about a different initial ROM, I think it's better not to point to `OneSidedIRKAReductor`. Just copying the point about initializing with interpolation points from `OneSidedIRKAReductor` is enough.

Reply:
So this point is resolved?